cli_reference/openshift_cli/getting-started-cli.adoc

```diff
@@ -16,6 +16,9 @@ You can install the OpenShift CLI (`oc`) either by downloading the binary or by
 // Installing the CLI by downloading the binary
 include::modules/cli-installing-cli.adoc[leveloffset=+2]
 
+// Installing the CLI by using the web console
+include::modules/cli-installing-cli-web-console.adoc[leveloffset=+2]
```

Comment:
This module should be broken up so that each procedure is its own module. I've noted that in the file itself. Once that's done, they can be added under this one as `[leveloffset=+3]` so that they still appear as subordinate to the **Installing the CLI by using the web console** heading.

Reply:
ack